tools/run_tests/helper_scripts/build_cxx.bat

```diff
@@ -30,13 +30,14 @@ If "%GRPC_BUILD_ACTIVATE_VS_TOOLS%" == "2019" (
 )
 
 If "%GRPC_CMAKE_GENERATOR%" == "Visual Studio 16 2019" (
-  @rem Force using a new-enough Windows 10 SDK that supports C++11's stdalign.h
-  @rem For some reason, cmake together with visual studio generator by default
+  @rem Always use the newest Windows 10 SDK available.
+  @rem A new-enough Windows 10 SDK that supports C++11's stdalign.h is required
+  @rem for a successful build.
+  @rem For some reason, by default cmake together with visual studio generator
```

Comment:
Nit: Can probably rephrase the "For some reason" bit now, since we know why it's happening. (It prefers the Windows 10 SDK whose version number happens to match the host OS's version number.)

Reply:
Done.